src/optable.ts

```diff
@@ -660,7 +556,7 @@ optable[opcodes.RAISE_VARARGS] = function(f: Py_FrameObject, t:Thread) {
         case 0:
             // todo: re-raising
             do_raise(f, t, cause, exc);
```

Comment:
```f.tryCatchException(t)``` does that currently.
Whereas ```thread.throwException``` outputs the traceback and terminates thread if an exception cannot be caught.